src/biomass_estimator_driver/scripts/biomass_estimator.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import json
+import os
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
+
+def biomassCorrelation(pix_count: float) -> float:
```

Comment:
should this be `snake_case`?

Reply:
Yep, this is part of Daniel's code, I will edit it to a standard code template and get back.

Reply:
Resolved #[6700027](https://github.com/precision-sustainable-ag/PlantMap3D-ROS/pull/38/commits/670002793c1804e2336a64a1605eebddef8bcfca)